app/views/layouts/_navigation_admin.html.haml

```diff
@@ -2,13 +2,13 @@
 
   - company = Company.main
 
-  %li.divider
-  %li= link_to 'Users', users_path
-  %li= link_to 'Security Roles', security_roles_path
-  %li= link_to 'Notifications', '/rad_common/notification_types'
-  %li= link_to 'Company Info', edit_company_path(company)
-  %li= link_to 'System Usage', '/rad_common/system_usages'
-  %li= link_to 'System Message', RadCommon::Engine.routes.url_helpers.new_system_message_path
-  %li= link_to 'Audit Search', audit_search_users_path
+  %li.dropdown-divider
+  %li= link_to 'Users', users_path, class: 'dropdown-item'
```

Comment:
Why do we have to add this class dropdown-item?

Reply:
https://getbootstrap.com/docs/4.0/migration/#dropdowns

"Dropdown items now require .dropdown-item" for styling